wpilibc/src/main/native/cpp/AnalogEncoder.cpp

```diff
@@ -12,6 +12,13 @@
 
 using namespace frc;
 
+AnalogEncoder::AnalogEncoder(int channel)
```

Comment:
Why can't we just call the constructor that takes a `std::shared_ptr<AnalogInput>` as we do in Java?